web-google-chat

tony1661 opened the pull request:
Adding icon for web-google-chat

Hokiper commented:
I think the centralization could be improved here.
My suggestion would be to do something like this (I took the orignal image from the app's official icon on Google Play):
![image](https://github.com/user-attachments/assets/f477a049-1a1d-423b-94c3-5c203a38cba0)

Also, it helps if you generate the bitmaps for the icons you make by running `src/render-apps-categories-bitmaps.py` and include them in the pull request. Remember to close all instances of Inkscape before running the script, otherwise you may end up with a bunch of windows.

I also suggest adding the following symbolic links to your icon:
`GoogleChat-chat.google.com.png`
`google-chat.png`

For that, you just have to add the following lines to `src/symbolic-apps-list` and run `src/create-symbolic-link-from-list.py`
![image](https://github.com/user-attachments/assets/4426a236-c9e6-4ead-bfd6-66599dfa95d6)

Hokiper commented:
You also need to unhide the `Baseplate` layer, edit the `icon-name` with the text tool so it matches the icon's name, and hide the `Baseplate` layer again. The script to generate the bitmaps won't work if the names don't match.
![image](https://github.com/user-attachments/assets/339c27e1-879f-4636-b157-62a07f0d7f72)

tony1661 commented:
Thank you @Hokiper for considering my contribution and for your assistance. I will make those changes.

Question: is there a guide I can follow going forward? I've contributed to [Delta Icons](https://github.com/Delta-Icons) before and they have a [contribution guide](https://github.com/Delta-Icons/android/blob/master/.github%2FCONTRIBUTING.md) that may help noobs like me 😅

Hokiper commented:
There's no written set of contribution guidelines, although this is actually something that has been talked about a few times - I even offered to make one myself.
The main discouragement (for me at least, since that's why I haven't made such a document yet) is that there's some uncertainty about if the Mint-Y icon theme will still be the default in the near future.

Hokiper commented:
But now that you mentioned it I'll bring that up with Clem just to see if he's still interested. If positive I should be making one on May's last week or so.

tony1661 commented:
Hi **@Hokiper** I have made the requested changes. Let me know if they are ok. if not I can adjust.


Hi **@mtwebster** I ran the script you mentioned however I am getting the following error:
```shell
$ src/render-apps-categories-bitmaps.py 
Traceback (most recent call last):
  File "/home/tfernandez/git/mint-y-icons/src/render-apps-categories-bitmaps.py", line 31, in <module>
    inkscape_version = str(subprocess.check_output(['inkscape', '-V'])).split(' ')[1].split('.')
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.12/subprocess.py", line 1955, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'inkscape'
```

Hokiper commented:
> Hi **@Hokiper** I have made the requested changes. Let me know if they are ok. if not I can adjust.

They look alright to me.

tony1661 commented:
Hi Team, just checking in if there are any other changes to be made?
